packages/checkout-ui-extensions/src/style/isEqual.ts

```diff
@@ -24,8 +24,8 @@ export function isEqual<
         );
       }
     } else {
-      const firstEntries = Object.entries(first!);
-      const secondEntries = Object.entries(second!);
+      const firstEntries = Object.entries(first);
+      const secondEntries = Object.entries(second);
```

Comment:
This concerns me a bit, did we forgot to add the nullish operator from checkout web or this is a mismatch ?

Reply:
We should keep them both in sync, yeah. It's weird that we need to add an exclamation point here and not in `checkout-web`, though 🤔